README.md

```diff
@@ -1,76 +1,84 @@
 
-### With docker (recommended\*)
+You can **additionally** provide a link to the source code for the project with `-sourceCodeUri` to enable additional analyzers such as the Vulnerability Scan.
+
+## Installation and usage
+
+### Docker (recommended)
+
+It is easiest to run the tool using the Docker image as it contains all the [security scanning tools](#security-tools) needed for the full set of analyzers - so you don't need to have these additional tools installed on your system.
 
 ```SHELL
-docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri [source_code_location/] [plugin_archive.zip]
+docker run --pull=always grafana/plugin-validator-cli [options] [path/to/plugin_archive.zip]
 ```
 
-Example:
+#### Example 1 (basic)
 
 ```SHELL
-docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
+docker run --pull=always grafana/plugin-validator-cli https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
 ```
 
-> \* Docker is recommended because the image contains all the [security scanning tools](#security-tools) for the validator
-
-### With NPX
+#### Example 2 (specifying source code location)
 
 ```SHELL
-npx -y @grafana/plugin-validator -sourceCodeUri [source_code_location/] [plugin_archive.zip]
+docker run --pull=always grafana/plugin-validator-cli -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
 ```
 
-Example:
+### NPX
 
 ```SHELL
-npx -y @grafana/plugin-validator -sourceCodeUri https://github.com/grafana/clock-panel/tree/v2.1.2 https://github.com/grafana/clock-panel/releases/download/v2.1.2/grafana-clock-panel-2.1.2.zip
+npx -y @grafana/plugin-validator@latest -sourceCodeUri [options] [path/to/plugin_archive.zip]
 ```
 
 ### Locally
 
-First you must compile and install it
+First you must compile and install it:
 
 ```SHELL
 git clone git@github.com:grafana/plugin-validator.git
 cd plugin-validator/pkg/cmd/plugincheck2
 go install
 ```
 
-Then you can run the utility
+Then you can run the utility:
 
 ```SHELL
 plugincheck2 -sourceCodeUri [source_code_location/] [plugin_archive.zip]
 ```
 
 ## Options
 
+Additional options can be passed to the tool:
+
 ```
 ❯ plugincheck2 -help
 Usage plugincheck2:
   -config string (optional)
         Path to configuration file
   -sourceCodeUri string (optional)
-        URI to the source code of the plugin. If set, the source code will be downloaded and analyzed. This can be a ZIP file URL, an URL to git repository or a local file (starting with `file://`)
+        URI to the source code of the plugin. If set, the source code will be downloaded and analyzed. This can be a ZIP file URL, a URL to git repository or a local file (starting with `file://`)
```

Comment:
minor edit (an is correct before URL)

an URL to a git repository, or a local file (starting with `file://`)

Reply:
I always debate this 😄 Google suggests _a_ URL (which does read more naturally imo) - [source1]( https://www.techtarget.com/whatis/feature/Which-is-correct-a-URL-or-an-URL#:~:text=Although%20the%20U%20at%20the,use%20a%20rather%20than%20an.) [source2](https://www.quora.com/Which-is-correct-A-URL-or-An-URL)